music21/musicxml/m21ToXml.py

```diff
@@ -2548,6 +2671,19 @@ def __init__(self,
         # has changed
         self.lastDivisions = None
 
+        # voiceNumberOffsetForMeasure is the offset that must be applied to any voice
+        # numbers automatically generated for the key measure in this part, in order
+        # to avoid voice numbers (either explicitly set, or automatically generated)
+        # in other parts in this staffGroup.  This is computed in a pre-pass over all
+        # the voices in all the measures in all the parts (in ScoreExporter).
+        self.voiceNumberOffsetForMeasure: t.Dict[stream.Measure, int] = {}
+
+        # The maximum voiceNum found in the voices in all the measures in this part.
```

Comment:
Isn't this no longer referring to "all the measures"?

Reply:
Yes.